components/brave_wallet_ui/components/buy-send-swap/swap-input-component/style.ts

```diff
@@ -1,10 +1,11 @@
 import styled from 'styled-components'
 import { CaratStrongDownIcon } from 'brave-ui/components/icons'
 import Refresh from '../../../assets/svg-icons/refresh-icon.svg'
+import ClipboardIcon from '../../../assets/svg-icons/clipboard-icon.svg'
 
 interface StyleProps {
   icon: string | undefined
-  isExchange: boolean
+  componentType: string
```

Comment:
Can we define `componentType: 'toAmount' | 'fromAmount' | 'toAddress' | 'buyAmount' | 'exchange' | 'selector'` somewhere so it's reusable here?

Reply:
Done! 😃
Created and exported a type definition called `BuySendSwapInputType` to be reused in the `style.ts`